app/views/dashboard/_content_views_widget.html.erb

```diff
@@ -0,0 +1,34 @@
+<h4 class="header">
+  <%= _("Content View History") %>
+</h4>
+
+<% organizations = Organization.current.present? ? [Organization.current] : User.current.allowed_organizations %>
+<% histories = Katello::ContentViewHistory.includes(:content_view_version, :task).in_organizations(organizations) %>
+<% histories = histories.readable.limit(Setting.per_page) %>
```

Comment:
The date also pushes the text off the widget, at least for me.  `mark_translated` is responsible for some of it, but maybe could we show the date in a more compact format?